application/controllers/dashboards.py

```diff
@@ -113,3 +114,20 @@ def dashboard_list(admin_client):
     return render_template('dashboards/index.html',
                            dashboards=dashboard_response,
                            **template_context)
+
+@app.route('{0}/<uuid>'.format(DASHBOARD_DELETE), methods=['GET', 'DELETE'])
+@requires_authentication
+@requires_feature('edit-dashboards')
```

Comment:
This feature should probably change to reflect the change in feature required for the button.